drf_user/managers.py

```diff
@@ -70,26 +92,22 @@ def create_superuser(self, username, email, password, name, mobile, **extra_fiel
         password: str
         name: str
         mobile: str
-        extra_fields: dict
+        kwargs: dict
 
         Returns
         -------
         User Instance
-
-        Author: Himanshu Shankar (https://himanshus.com)
         """
         vals = update_user_settings()
 
-        extra_fields.setdefault("is_superuser", True)
-        extra_fields.setdefault("is_staff", True)
-        extra_fields.setdefault("is_active", vals.get("DEFAULT_ACTIVE_STATE", False))
+        kwargs.setdefault("is_superuser", True)
+        kwargs.setdefault("is_staff", True)
+        kwargs.setdefault("is_active", vals.get("DEFAULT_ACTIVE_STATE", False))
```

Comment:
What does the Python convention talk about modifying kwargs?

Reply:
https://stackoverflow.com/questions/45883655/is-it-always-safe-to-modify-the-kwargs-dictionary